charts/spire/charts/spiffe-csi-driver/templates/scc-restricted-csi.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if and .Values.global.openshift .Values.restrictedScc.enabled }}
+{{- $name := default (printf "restricted-%scsi" (.Values.restrictedScc.v2 | ternary "v2-" "")) .Values.restrictedScc.name -}}
+{{- $v2Description := "This is the most restrictive SCC and it is used by default for authenticated users. On top of the legacy 'restricted' SCC, it also requires to drop ALL capabilities and does not allow privilege escalation binaries. It will also default the seccomp profile to runtime/default if unset, otherwise this seccomp profile is required." -}}
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+allowHostPorts: false
+priority: null
+requiredDropCapabilities:
+{{- if .Values.restrictedScc.v2 }}
+  - ALL
+{{- else }}
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
+{{- end }}
+allowPrivilegedContainer: false
+runAsUser:
+  type: MustRunAsRange
+users: []
+allowHostDirVolumePlugin: false
+{{- if .Values.restrictedScc.v2 }}
+seccompProfiles:
+  - runtime/default
+{{- end }}
+allowHostIPC: false
+seLinuxContext:
+  type: MustRunAs
+readOnlyRootFilesystem: false
+metadata:
+  annotations:
+    kubernetes.io/description: >-
+      {{ $name }} denies access to all host features and requires pods to be
+      run with a UID, and SELinux context that are allocated to the namespace.
+      {{ .Values.restrictedScc.v2 | ternary $v2Description "" }}
+  name: {{ $name }}
```

Comment:
metadata.name being near the top in the standard ordering really helps us users when we are looking at what everything does.